melosys-eessi-app/src/main/resources/application.yml

```diff
@@ -28,6 +28,11 @@ spring:
       sasl.jaas.config: org.apache.kafka.common.security.plain.PlainLoginModule required username="${melosys.systemuser.username}" password="${melosys.systemuser.password}";
       reconnect.backoff.ms: 1000
 
+  servlet:
+    multipart:
```

Comment:
Jeg ville sette et større tall som f.eks. 500MB eller undersøke søknadenes størrelse.

Reply:
Ja, jeg er enig. Skal se om jeg kan finne ut av det.

Når jeg kjører lokalt, får jeg faktisk opp størrelse på fila:

```
08:20:20.843 ERROR n.n.m.e.c.RestExceptionHandler - Feil oppstått: 
org.springframework.web.multipart.MaxUploadSizeExceededException: Maximum upload size exceeded; nested exception is java.lang.IllegalStateException: org.apache.tomcat.util.http.fileupload.impl.SizeLimitExceededException: the request was rejected because its size (10488836) exceeds the configured maximum (2097152)
```

Kanskje noen forskjeller i versjoner, miljø eller config som gjør at dette ikke logges i prod.

Reply:
😮 ❓

Reply:
10-20MB føler jeg burde holde i massevis. Jeg stusser litt om vi sender en søknad-pdf som er mye større enn dette her 😅  
Arild mener forresten grensen i RINA er 100MB.

Reply:
Øker maks til 10MB og legger til logging i navikt/melosys-api#190.